src/rust/src/pkcs12.rs

```diff
@@ -521,6 +535,33 @@ fn serialize_bags<'p>(
     Ok(pyo3::types::PyBytes::new(py, &asn1::write_single(&p12)?))
 }
 
+#[pyo3::pyfunction]
+#[pyo3(signature = (certs, encryption_algorithm))]
+fn serialize_java_truststore<'p>(
+    py: pyo3::Python<'p>,
+    certs: pyo3::Bound<'_, pyo3::PyAny>,
```

Comment:
It might be possible to just accept a `Vec<Py< PKCS12Certificate>>`? (If that's an error, ignore this suggestion :-))

Reply:
Looks like it works! That's much cleaner. Updated.